sgkit/api.py

```diff
@@ -24,7 +24,6 @@ def create_genotype_call_dataset(
     variant_id: Any = None,
 ) -> xr.Dataset:
     """Create a dataset of genotype calls.
-
```

Comment:
We should probably put  any docstring formatting changes in a separate PR/issue.